resources/home/dnanexus/tests/test_generate_bed.py

```
@@ -301,3 +301,168 @@ def test_transcript_missing_from_exons(
+	            "HGNC:44506",
+	            "HGNC:93010"],
+        genome_build="_b38.bed",
+        output_prefix=f"{TEST_DATA_DIR}/test_beds/R50.1_Early_onset_dementia",
+        flank=None,
+        additional_regions=None)
+
+       
+        # make expected bed file and save as test_expected_bed_file.bed
+        
+        columns=["chromosome","start","end","transcript"]
+        expected_bed_file= pd.DataFrame(
```

Ruff (0.15.12)

[error] Undefined name `pd`

(F821)